src/reanimated2/animation/util.ts

```diff
@@ -270,6 +297,17 @@ function decorateAnimation<T extends AnimationObject | StyleLayoutAnimation>(
     timestamp: Timestamp,
     previousAnimation: Animation<AnimationObject>
   ) => {
+    if (animation.reduceMotion === undefined) {
+      animation.reduceMotion = shouldReduceMotion('system');
+    }
+    if (animation.reduceMotion) {
+      animation.current = animation.toValue;
```

Comment:
Did you tested colors animations?

Reply:
I tested the Animate Colors example and it worked fine